python/desc/imsim/atmPSF.py

```diff
@@ -14,7 +14,8 @@
 class OptWF(object):
     def __init__(self, rng, wavelength, gsparams=None):
         u = galsim.UniformDeviate(rng)
-        self.deviations = mock_deviations(seed=int(u()*2**31))
+        deviationsFudgeFactor = 3.0
```

Comment:
I think it would be good to put a comment above the definition of `deviationsFudgeFactor`  pointing to the GitHub thread that led to its creation and  briefly saying the nature of this fudge factor (i.e., that it was based on empirical attempts to get reasonable optical PSF shape given that our model was based on a very small number of sims that  don't span the space expected in reality).  This will help with any inevitable future confusion over where this came from etc.